app/views/teachers/classrooms/students/edit_quiz_score.html.slim

```diff
@@ -0,0 +1,20 @@
+.row
+  .small-12.columns
+    .center.paper
+      .row
+        .small-12.columns
+          h2 = t ".heading", student_email: @student.email
+          .subtitle = t ".subtitle"
```

Comment:
If the `subtitle` is blank, why not just delete this line and remove the subtitle key from `en.yml`?